middle_end/flambda2/lattices/or_bottom.mli

```diff
@@ -36,4 +36,8 @@ module Let_syntax : sig
   val ( let<* ) : 'a t -> ('a -> 'b t) -> 'b t
 
   val ( let<+ ) : 'a t -> ('a -> 'b) -> 'b t
+
+  (** [let<+$] returns the input ['a t] if the mapping function returns a value
+      that is physically-equal to its input. *)
+  val ( let<+$ ) : 'a t -> ('a -> 'a) -> 'a t
```

Comment:
This operator is not even used. Do we really need it ?

Reply:
I thought it might be better to have it for consistency, but happy to remove it.